ISSUE_TEMPLATE.md

```diff
@@ -1,40 +1,49 @@
-<!--
```

Comment:
Shouldn't the `ISSUE_TEMPLATE.md` be removed completely?
It already exists a [template](https://github.com/pulsar-edit/.github/blob/main/.github/ISSUE_TEMPLATE/bug-report.yml) for bug reports.

Reply:
Yup, great point. I think this PR was made before that become our initiative. Thanks for reminding me.